pkg/machines/components/storagePools/storagePoolOverviewTab.jsx

```diff
@@ -32,16 +32,26 @@ export const StoragePoolOverviewTab = ({ storagePool }) => {
 
     return (
         <div className='ct-form-layout'>
-            {  poolSource && poolSource.host && <React.Fragment>
+            { poolSource && poolSource.host && <React.Fragment>
                 <label className='control-label' htmlFor={`${idPrefix}-host`}> {_("Host")} </label>
-                <div id={`${idPrefix}-host`}
+                <div id={`${idPrefix}-host`}>
```

Comment:
Please move that fix into the previous commit, so that this doesn't introduce a broken state.

Reply:
Done